addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -96,6 +96,7 @@ def _compute_l10n_latam_amount_and_taxes(self):
         recs_invoice = self.filtered(lambda x: x.is_invoice())
         for invoice in recs_invoice:
             tax_lines = invoice.line_ids.filtered('tax_line_id')
+            currencies = invoice.line_ids.mapped('currency_id')
```

Comment:
@nicomacr There should only be one currency on an invoice, so it would be logical to map the one on the invoice itself

Reply:
Yes, I fixed this as you said.

Reply:
@jco-odoo I'm using the same approach odoo did [here](https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L1126)